packages/components/src/circular-option-picker/style.scss

```diff
@@ -67,8 +67,8 @@ $color-palette-circle-spacing: 12px;
 .components-circular-option-picker__option {
 	display: inline-block;
 	vertical-align: top;
-	height: 100%;
-	width: 100%;
+	height: 100% !important;
```

Comment:
Unfortunate override. Needs some kind of unstyled Button variant.

Reply:
Yeah, quite a bummer with these overrides. I think it would make sense to explore an unstyled variant at some point. Perhaps opening an issue is a good first step to keep it documented?

Reply:
Done ✅  #67320